control_panel_api/tests/test_models.py

```diff
@@ -0,0 +1,79 @@
+from django.test import TestCase
+
+from control_panel_api.models import (
+    Role,
+    Team,
+    TeamMembership,
+    User,
+)
+
+
+class FixtureTestCase(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
Should this be called `setUpClass`? https://docs.python.org/3.6/library/unittest.html#setupclass-and-teardownclass

Reply:
Bear in mid that `TestCase` here is an instance of `django.test.TestCase` which provides the`setUpTestData` method for this specific use case: https://docs.djangoproject.com/en/1.11/topics/testing/tools/#django.test.TestCase.setUpTestData

Reply:
👍